0.6.5 does not honor tab size settings

## editorconfig-vscode version: 0.6.5

### Expected behavior

Tab size settings applied to editor in workspace

### Actual behavior

Invalid tab size in editor

### Steps to reproduce the behavior

```ini
root = true

[*]
indent_style = space
indent_size = 4

[*.{c,cpp,h,hpp,proto}]
indent_style = tab
indent_size = 2
```

`indent_style` is `tab` but I still see `indent_size = 4` in editor:

![zrzut ekranu z 2017-04-10 09-30-12](https://cloud.githubusercontent.com/assets/141999/24850428/5b203780-1dd0-11e7-8b97-d99c5fb20b78.png)

Hi, @jedmao. In my case, there is no any output (no EditorConfig option in the output window). I think the extension totally fails to start.
I also found that there are some error messages at the developer tools:
```
[Extension Host] Activating extension `EditorConfig.EditorConfig` failed: Cannot find module './InsertFinalNewline'.
[Extension Host] Activating extension `EditorConfig.EditorConfig` failed:  Cannot find module './InsertFinalNewline'
[Extension Host] Here is the error stack:  Error: Cannot find module './InsertFinalNewline'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Function.J.i._load (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:4:374825)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/bist/.vscode/extensions/EditorConfig.editorconfig-0.7.0/out/src/transformations/index.js:2:28)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Function.J.i._load (/usr/share/code/resources/app/out/vs/workbench/node/extensionHostProcess.js:4:374825)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/bist/.vscode/extensions/EditorConfig.editorconfig-0.7.0/out/src/DocumentWatcher.js:14:27)
    at Module._compile (module.js:556:32)
Activating extension `EditorConfig.EditorConfig` failed: Cannot find module './InsertFinalNewline'.
There was an error in running the linter 'prospector'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+## 0.9.1
+- Fix [issue 135](https://github.com/editorconfig/editorconfig-vscode/issues/135): extension does not load on Linux systems, due to case sensitivity.
+
 ## 0.9.0
 - Improve output channel messaging.
 

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "displayName": "EditorConfig for VS Code",
   "description": "EditorConfig Support for Visual Studio Code",
   "publisher": "EditorConfig",
-  "version": "0.9.0",
+  "version": "0.9.1",
   "icon": "EditorConfig_icon.png",
   "engines": {
     "vscode": "^1.11.0"

